website_data.json

```diff
@@ -1,5 +1,10 @@
 {
 	"factoids": [
+		{		
+			"banner": "Be kind to people, stop racism and discrimination. ",
+			"id": "1",
```

Comment:
Kindly change the "id" to some other value (because the list now contains 2 JSON objects with same id=1) and preferably at the end of the "factoids" list.